medicines/search/src/main.rs

```diff
@@ -18,7 +18,7 @@ async fn main() {
             .expect("Failed to delete datasource"),
         ("create_index", Some(m)) => {
```

Comment:
I think it would make sense to update the command to be `create_or_update_index` too, updating the `cli.yaml` and the `README.md` accordingly, but get that we're working to a tight schedule so will approve these changes and leave it to you to decide.

Reply:
Good shout, will update that.